core/model/src/main/java/org/eclipse/rdf4j/model/impl/SimpleValueFactory.java

```diff
@@ -147,25 +129,14 @@ public Triple createTriple(Resource subject, IRI predicate, Value object) {
 	/**
 	 * Generates a new bnode prefix and resets <var>nextBNodeID</var> to <var>1</var> .
 	 */
+	@Deprecated(since = "4.0.0", forRemoval = true)
```

Comment:
Given this is scheduled for 4.0, and is protected, I am fine with just immediately removing it as well.